source/less/blocks/after-menu.less

```
@@ -1,16 +1,28 @@
 .after-menu {
 
   font-family: "Oswald", "Tahoma", sans-serif;
-  background: url("../img/index-background-mobile@1x.jpg") center no-repeat;
-  background-size: cover;
-  width: 100%;
-  height: 50vh;
+  background: url("../img/index-background-mobile@1x.jpg");
+  background-size: 320px 305px, 100%;
+  background-position: top center;
+  background-repeat: no-repeat;
   color: @white;
-  align-items: center;
+  display: flex;
+  text-align: center;
   justify-content: space-between;
   position: relative;
   padding: 27px 36px;
 
+  @media (min-width: 768px), (max-width: 1440px) {
+    background: none;
+    color: @black;
+    align-items: flex-start;
+    justify-content: start;
+  }
+
+  @media (max-width: 1440px) {
```

```suggestion
  @media (min-width: 1440px) {
```

И в остальных случаях (там, где используете `max-width`) - `min-width`.